settings.gradle

```diff
@@ -1 +1,12 @@
+plugins {
+    id "com.gradle.enterprise" version "3.5"
+}
+
+gradleEnterprise {
+    buildScan {
+        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceAgree = "yes"
```

Comment:
We should add something like this: 
`isUploadInBackground = System.getenv("CI") == null`

https://docs.gradle.com/enterprise/gradle-plugin/#configuring_background_uploading